dashboards/erlang/README.md

```diff
@@ -0,0 +1,23 @@
+# Erlang magic dashboard
+
+AppSignal tracks Erlang VM metrics.
+It has graphs metrics on IO, schedulers, processes and memory, to give a high-level overview of what's happening on the VM.
+
+## IO
+
+This graph shows the amount of input and output you have cumulatively, expressed in kilobytes.
```

Comment:
The graph doesn't use the file size format, should it?

https://github.com/appsignal/public_config/blob/e7aeacf2218cc9eb8952b4c6934c7021e3582ee0/dashboards/erlang/main.json#L14

Reply:
Ah, yes, good call. This number is in kilobytes.